### PR TITLE
enhanceMent: Add Star the Repository button for Meshery

### LINKED ISSUE #4361: Add "Star the Repository" button for Meshery
Add a button on the  [meshery page](https://layer5.io/cloud-native-management/meshery) to redirect the user to the meshery Repository 

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by tarunsamanta2k20**
**Description**

This PR fixes #4361 

**Notes for Reviewers**

Add a button on the [meshery page](https://layer5.io/cloud-native-management/meshery) to redirect the user to the meshery Repository
**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 
Add a button on the [meshery page](https://layer5.io/cloud-native-management/meshery) to redirect the user to the meshery Repository

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit ecdb0a062fae3d63131cec05679a45cc78f1fb7f at: https://6484bb33d2a28e07704b0e12--layer5.netlify.app

**Comment by vishalvivekm** [comment]
@tarunsamanta2k20 Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

**Review by suhail34 — CHANGES_REQUESTED** [review]
It would be good if the github repo opens up on a new tab rather than on same tab

**Comment by l5io** [comment]
🚀 Preview for commit 78566384b73fba6884ea5ca2632a2ef20b728dbe at: https://64861760f1e5e14625fd9715--layer5.netlify.app

**Review by suhail34 — CHANGES_REQUESTED** [review]
The buttons in the left and right don't align horizontally and after clicking on Star the repository its background color becomes gray in color if you could fix this too it would be nice 
image

**Comment by tarunsamanta2k20** [comment]
> The buttons in the left and right don't align horizontally and after clicking on Star the repository its background color becomes gray in color if you could fix this too it would be nice image

@suhail34 button color is changing because this is the default behavior. It's the same for all the other buttons as well. you may try opening another button on a new tab as well.

**Comment by tarunsamanta2k20** [comment]
@suhail34 is it looking good?
image

**Review by suhail34 — APPROVED** [review]
LGTM Good work

**Review by hirentimbadiya — APPROVED** [review]
Looks Good @tarunsamanta2k20 👍🏻

**Comment by thisiskaransgit** [comment]
@Chadha93, Do we need the button as meshery.io?

**Comment by Chadha93** [comment]
> @Chadha93, Do we need the button as meshery.io?

@thisiskaransgit, 
 "as meshery.io"? - Nope.

File: src/sections/Meshery/index.js [hunk 1]
```diff
@@ -29,20 +30,24 @@ const MesheryPage = () => {
...
+                <GiClockwork size={21} className="button-icon" />
+              </Button>
             </Col>
-            <Col lg={6} md={6} className="meshery-hero-img">
+            <Col lg={6} md={6} className="meshery-hero-img desc-text">
               <video autoPlay muted loop controls className="meshery-video">
                 <source src={mesheryDemo} type="video/mp4" />
               </video>
               {/* <img className="meshery-sup-img" src={mesheryFullStack} alt="Meshery the multi-mesh manager" /> */}
+              <Button primary className="banner-btn" style={{ "margin-top": "17px", "background-color": "#EBC017" }}
```
Comment (Chadha93) [inline src/sections/Meshery/index.js:47]: @tarunsamanta2k20 Please refer to the theme, when adding colors, and void explicit margin, use flex, grid, and padding... 
`margin-top` here seems fragile.

Comment (tarunsamanta2k20) [reply src/sections/Meshery/index.js:47]: Have you read the above comments? 
The requirements are the following:

1. The new button should open in a new tab.
2. After clicking on the new button, the button color shouldn't be changed.
3. The new button should be at the same level as the other two buttons.

What would be your suggestion other than this?

Comment (tarunsamanta2k20) [reply src/sections/Meshery/index.js:47]: If I remove the CSS it'll look like this.
image

Comment (tarunsamanta2k20) [reply src/sections/Meshery/index.js:47]: If I apply padding from the top the the whole button width is getting increased.

Comment (vishalvivekm) [reply src/sections/Meshery/index.js:47]: Let's discuss this in websites call. Add this as agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) @tarunsamanta2k20

**Comment by tarunsamanta2k20** [comment]
@suhail34 @Chadha93 @vishalvivekm is it looking more good?

image

**Comment by l5io** [comment]
🚀 Preview for commit b19adb8b4942b6d20aaa0538334491cc233a6298 at: https://649eb7fc39c2256490349fec--layer5.netlify.app

**Comment by tarunsamanta2k20** [comment]
@aishwarya281098 here

File: src/sections/Meshery/meshery.style.js [hunk 2]
```diff
@@ -43,7 +43,11 @@ const MesheryWrapper = styled.section`
                      
                     .button-icon{
                         margin-right: 0.625rem;
-                    }    
+                    }   
+                }
+                .align-btn{
+                    bottom: -12px;
+                    background-color: #EBC017;
```
Comment (Chadha93) [inline src/sections/Meshery/meshery.style.js:50]: @tarunsamanta2k20 Please refer theme to add color.

Comment (tarunsamanta2k20) [reply src/sections/Meshery/meshery.style.js:50]: @Chadha93 the button primary is using this color code.

Comment (Chadha93) [reply src/sections/Meshery/meshery.style.js:50]: Yes it is, I meant to convey -  use theming like this - `${props => props.theme.blackToWhite}; `  to add color, (note this is an example, refer to the theme reference which has the desired color).

Comment (tarunsamanta2k20) [reply src/sections/Meshery/meshery.style.js:50]: @Chadha93 got it.

**Comment by l5io** [comment]
🚀 Preview for commit 613e498f60bf813b32f503be053b574df022c839 at: https://64a1d45f6243bb038740aff1--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 94160da29373f2921887e116f2c70b8ce3c1fda3 at: https://64a1d922967a46090358ebd5--layer5.netlify.app

**Comment by tarunsamanta2k20** [comment]
@Chadha93 if everything has been done for this PR, then please merge. if any changes are required then please let me know.

File: src/sections/Meshery/index.js [hunk 3]
```diff
@@ -38,11 +39,15 @@ const MesheryPage = () => {
                 <FiDownloadCloud size={21} className="button-icon" />
               </Button>
             </Col>
-            <Col lg={6} md={6} className="meshery-hero-img">
-              <video autoPlay muted loop controls className="meshery-video">
+            <Col lg={6} md={6} className="meshery-hero-img desc-text">
+              <video autoPlay muted loop controls className="meshery-video" >
```
Comment (thisiskaransgit) [inline src/sections/Meshery/index.js:43]:
```suggestion
            <Col lg={6} md={6} className="meshery-hero-img">
              <video autoPlay muted loop controls className="meshery-video">
```

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge, thank you for your contribution :tada: 
// @Shivam-AfA

File: src/sections/Meshery/index.js (hunk 3 above)
Comment (tarunsamanta2k20) [reply src/sections/Meshery/index.js:43]: If I remove the css `desc-text` then the button won't look like the other two button.

**Comment by Shivam-AfA** [comment]
Thanks @tarunsamanta2k20 for this.

**Pull request merged**